libcxx/test/support/test_range.h

```diff
@@ -89,4 +91,12 @@ concept simple_view =
     std::same_as<std::ranges::iterator_t<Range>, std::ranges::iterator_t<const Range>> &&
     std::same_as<std::ranges::sentinel_t<Range>, std::ranges::sentinel_t<const Range>>;
 
+template <class T, class U = T>
```

Comment:
Optional: perhaps add a comment like `// [concept.equalitycomparable]` to indicate that this concept comes from the Standard (where it's exposition-only so we can't use it directly)?

Reply:
I believe that I addressed this with ee61f15cfef3fd8b08d818d7455c2192d94980ea.